website/static/img/showcase/vivo.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="104px" height="28px" viewBox="0 0 103 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g id="海外一期" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
What's this?

Reply:
I copied this from [http://vivo.com/fr](http://vivo.com/fr). Fixed, thanks.